tools/encodings/GF Glyph Sets/Cyrillic/GF-cyrillic-pro_unique-glyphs.nam

```diff
@@ -9,6 +10,8 @@
 0x0495  ҕ uni0495
 0x049E  Ҟ uni049E
 0x049F  ҟ uni049F
+0x04A6  Ҧ CYRILLIC CAPITAL LETTER PE WITH MIDDLE HOOK
```

Comment:
@alexeiva What about these in Cyrillic-Plus:
```
0x0494  Ҕ CYRILLIC CAPITAL LETTER GHE WITH MIDDLE HOOK
0x0495  ҕ CYRILLIC SMALL LETTER GHE WITH MIDDLE HOOK
```

I also added them from the filter-lists.